jenkinsfiles/grant_access_to_app.groovy

```diff
@@ -1,28 +1,41 @@
 node {
 
+    properties([
+        parameters([
+            string(
+                name: 'APP_NAME',
+                description: 'Shiny App name',
+                defaultValue: ''),
+            string(
+                name: 'EMAILS',
+                description: 'Semicolon delimited list of email addresses',
+                defaultValue: ''),
+        ])
+    ])
+
     stage ("Git checkout") {
         checkout scm
     }
 
     // Creates passwordless user in Auth0 and adds to app group
     // NOTE: Groups provided by Auth0 Authorization Extension
     stage ("Create Auth0 passwordless user and add to app group") {
+      env.AUTHZ_API_ID = 'urn:auth0-authz-api'
       withCredentials([
-        usernamePassword(
-            credentialsId: 'auth0-mgmt-api',
-            usernameVariable: 'CLIENT_ID',
-            passwordVariable: 'CLIENT_SECRET'),
         string(
-            credentialsId: 'authz_api_url',
-            variable: 'AUTHZ_API_URL'),
+            credentialsId: 'AUTH0_CLIENT_ID',
```

Comment:
Feel free to change if you want. The only reason I used the more generic "Secret Text" was because I struggled to find documentation for the jenkins credentials plugin. If we think CLIENT_ID/CLIENT_SECRET as username/password so maybe the `usernamePassword` credential type may be more appropriate. If.